src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -84,25 +105,73 @@ std::optional<std::array<double, Dim>> Translation<Dim>::inverse(
              << function_of_time.size()
              << ") does not match the dimension of the translation map (" << Dim
              << ").");
-  double radial_function_value = 1.0;
-  if (f_of_r_ != nullptr) {
-    const double root = root_finder(target_coords - center_, function_of_time);
-    radial_function_value = (*f_of_r_)(root);
-  }
-  for (size_t i = 0; i < Dim; i++) {
-    gsl::at(result, i) -= function_of_time[i] * radial_function_value;
+  if (std::isnan(inner_radius_)) {
```

Comment:
Can you add some code comments for all these `if` statements saying what is actually happening? Like "Math function specific. doing a root find" or "inner/outer radius specific". That way it's a bit easier for devs to parse all the `if`s and `for`s